context/app/static/js/components/detailPage/ContributorsTable/utils.ts

```diff
@@ -47,65 +61,79 @@ export interface Contributor {
   orcid: string;
 }
 
+export interface Contact extends Contributor {
+  isContact: true;
+}
+
 const humanBooleanToBoolean = (humanBoolean: LegacyBoolean | CEDARBoolean): boolean =>
   humanBoolean === 'TRUE' || humanBoolean === 'Yes';
 
-const normalizeLegacyContributor = (contributor: LegacyContributor): Contributor => ({
+export const normalizeContributor = (contributor: ContributorAPIResponse): Contributor => ({
   affiliation: contributor.affiliation,
-  name: contributor.name,
-  email: 'N/A',
+  name: 'name' in contributor ? contributor.name : contributor.display_name,
```

Comment:
```suggestion
  name: contributor?.name ?? contributor.display_name,
```
This would have the same effect as most of these ternaries. Resolves to the lefthand side if it is defined else the right hand side. Not a blocking issue, but worth knowing.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing